x-pack/plugins/actions/server/builtin_action_types/jira/service.ts

```diff
@@ -102,10 +102,14 @@ export const createExternalService = (
     return fields;
   };
 
-  const createErrorMessage = (errorResponse: ResponseError | null | undefined): string => {
+  const createErrorMessage = (errorResponse: ResponseError | string | null | undefined): string => {
     if (errorResponse == null) {
       return '';
     }
+    if (typeof errorResponse === 'string') {
+      // Jira error.response.data can be string!!
```

Comment:
@cnasikas not sure what the response was looking like for other errors, but for 401 data was a string

Reply:
@stephmilovic Nice catch! In their [API](https://developer.atlassian.com/cloud/jira/platform/rest/v2/api-group-issues/#api-rest-api-2-issue-post) it says that they return an `ErrorCollection` for 400, 401, 403